utils/doclint/preprocessor/index.js

```diff
@@ -34,9 +36,6 @@ module.exports.ensureReleasedAPILinks = function (sources, version) {
 };
 
 module.exports.runCommands = function (sources, version) {
-  // Release version is everything that doesn't include "-".
-  const isReleaseVersion = !version.includes('-');
```

Comment:
Hmm, actually I think this would now cause problems on CI. We need to keep this check and combine it with the new environment variable. Will fix.

Reply:
Done